extensions/wikia/EditPageLayout/css/EditPageLayout.scss

```diff
@@ -599,3 +599,9 @@ body {
 		@include breakpoints-large-typography;
 	}
 }
+
+#InfoboxPreview {
+	margin-left: 50px;
+	margin-top: 10px;
+	width: 150px;
+}
```

Comment:
I agree with @RafLeszczynski about not using an id to style this. If the style declaration needs to be more specific you can use something like `input.InfoboxPreview` or consider nesting this under `.EditPageRail`.
Also, you can center this horizontally using:
```
.EditPageRail .InfoboxPreview {
  margin: 10px auto 0;
  display: block;
}
```
Instead of the 50px magic number